plugins/CoreAdminHome/vue/src/BrandingSettings/BrandingSettings.vue

```diff
@@ -255,11 +262,23 @@ export default defineComponent({
     },
     save() {
       this.isLoading = true;
+      AjaxHelper.post({
+        module: 'API',
+        method: 'CoreAdminHome.setBrandingSettings',
+      },
+      {
+        useCustomLogo: this.enabled ? '1' : '0',
```

Comment:
Something in the error handling feels a bit weird, don't know if we should ignore that edge case.

If you upload an invalid file, you get a message saying "the uploaded file couldn't be processed" (can be tested with a plain text file). The UI will only show that error message and not change the preview images of the already uploaded custom logo/favicon.

However, after you have seen that error, hitting save will deactivate the custom logo setting.

Reply:
Thanks Marc, will add that to the UI test and fix that usecase.